css/style-middle-section.css

```diff
@@ -0,0 +1,192 @@
+@media (min-width: 375px) {
+  #works {
+    /* 375 x 2788 */
+    background-color: #e5e5e5;
+    display: grid;
+    grid-gap: 88px;
+    padding: 114px 24px 114px 24px;
+  }
+
+  .card-work {
+    border-radius: 16px;
+    padding: 16px 16px 28px 16px;
+    gap: 12px;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .first-text-section h1 {
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    font-weight: 700;
+  }
+
+  .frame {
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    margin-top: 12px;
+  }
+
+  .frame1 {
+    color: #344563;
+    font-style: normal;
+  }
+
+  .dot {
+    margin: 0 12px 0 12px;
+  }
+
+  .frame2 {
+    color: #7a869a;
+  }
+
+  .frame3 {
+    color: #7a869a;
+  }
+
+  .primary-text {
+    font-size: 15px;
+    line-height: 24px;
+    font-style: normal;
+    font-weight: 400;
+    color: #344563;
+    margin-top: 12px;
+  }
+
+  .tags {
+    margin-top: 12px;
+  }
+
+  .action a {
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    font-style: normal;
+    display: flex;
+  }
+
+  .skills-li a {
+    display: flex;
+    justify-content: space-between;
+    height: 72px;
+    align-items: center;
+  }
+
+  .tags li a {
+    font-size: 12px;
+    line-height: 16px;
+    font-weight: 500;
+    font-style: normal;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    background-color: #ebebff;
+    border-radius: 8px;
+    padding: 4px 12px 4px 12px;
+    margin-right: 8px;
+  }
+
+  .action {
+    width: 126px;
+    height: 48px;
+    border: 1px solid #396df2;
+    color: #396df2;
+    border-radius: 8px;
+    margin-top: 24px;
+    padding: 12px;
+  }
+
+  .action:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .action:disabled {
+    background-color: #e5e5e5;
+    color: #5e6c84;
+    border: 1px solid #c1c7d0;
+  }
```

Comment:
- [ ] Excellent effort in applying the styles for `hover`, and `disbaled` state to the button. 😃👍🙌 However, please apply style for `pressed` state as well, so that it will be interactive for the user when he clicks the button. 😊 You can achieve this by applying a pseudo class of `:active` to the button. 
- [ ] Please add an `enabled` state to your button as mentioned in the design guidelines in Figma, and as requested by the previous reviewer [here](https://github.com/BiancaMe/Portfolio-Mobile/pull/3#discussion_r1224544574), so that it can be used in combination with the `:disabled` pseudo class, and to style enabled and disabled form elements differently.